stage1/init/init.go

```diff
@@ -126,17 +126,25 @@ var (
 	debug       bool
 	privNet     common.PrivateNetList
 	interactive bool
+	mdsToken    string
+	localhostIP net.IP
 )
 
 func init() {
 	flag.BoolVar(&debug, "debug", false, "Run in debug mode")
 	flag.Var(&privNet, "private-net", "Setup private network")
 	flag.BoolVar(&interactive, "interactive", false, "The pod is interactive")
+	flag.StringVar(&mdsToken, "mds-token", "", "MDS auth token")
```

Comment:
`--debug` and `--private-net` are documented in [Stage1 ACI implementor's guide](https://github.com/coreos/rkt/blob/master/Documentation/devel/stage1-implementors-guide.md#arguments) but `--mds-token` is missing. It would be useful to document for alternative stage1s such as the lkvm stage1 (#946 #1011).

@iaguis: I notice that `--interactive` was forgotten as well when it was introduced.